charts/csm-authorization/values.yaml

```diff
@@ -57,16 +57,21 @@ authorization:
   storageCapacityPollInterval: 5m
 
 redis:
+  name: redis-csm
+  sentinel: sentinel
+  rediscommander: rediscommander
+  replicas: 5
   images:
-    redis: redis:6.0.8-alpine
+    redis: redis
     commander: rediscommander/redis-commander:latest
-
-  # set the storageClass for redis to use. otherwise, the default storage class is used
-  # storageClass: local-storage
+    
+  # by default, csm-authorization will deploy a local (https://kubernetes.io/docs/concepts/storage/storage-classes/#local) volume for redis
+  # to use a different storage class for redis, uncomment the following line and specify the name of the storage class
+  # NOTE: the storage class must NOT be a storage class provisioned by a CSI driver to be configured with this instance of CSM Authorization
```

Comment:
Are these comments accurate with `emptyDir`?

Reply:
Will fix

Reply:
Removed the comment, not applicable anymore